apps/docker-base/Makefile.docker-base

```diff
@@ -1,35 +1,40 @@
-# -*- makefile -*-
+# Makefile for Docker image setup
+
 help:
 	@echo "This should be run as part of the Dockerfile"
 	false
 
 user-setup:
+	# Create app group and user
 	groupadd --gid 1000 app
 	useradd -d /app --uid 1000 --gid app app
 	chown -R app:app /app
 
 apt-setup:
+	# Configure APT to keep downloaded packages
 	rm -f /etc/apt/apt.conf.d/docker-clean
 	echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
 
 apt-install:
-	DEBIAN_FRONTEND=noninteractive apt update
-	# gcc and python3-dev needed on arm for guidance
-	DEBIAN_FRONTEND=noninteractive apt -y install --no-install-recommends python3-poetry gcc python3-dev
+	# Install necessary packages without recommended packages
+	DEBIAN_FRONTEND=noninteractive apt update && \
+	DEBIAN_FRONTEND=noninteractive apt -y install --no-install-recommends python3-poetry gcc python3-dev && \
+	# Clean up to reduce image size
```

Comment:
We don't want this because of the caching.   From docker-base/Dockerfile.buildx
```
RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
    --mount=type=cache,target=/var/lib/apt,sharing=locked \
    make -f Makefile.docker-base apt-install
```
The first two lines are caching the directories, so they don't show up in the image.

You can verify that on the existing containers:
```
% docker run -it -u root arynai/sycamore-base bash
root@9983ae28df15:/app# find /var/cache/apt
/var/cache/apt
/var/cache/apt/archives
/var/cache/apt/archives/lock
/var/cache/apt/archives/partial
root@9983ae28df15:/app# find /var/lib/apt
/var/lib/apt
/var/lib/apt/mirrors
/var/lib/apt/mirrors/partial
/var/lib/apt/periodic
/var/lib/apt/extended_states
/var/lib/apt/lists
```